core/src/mindustry/world/blocks/defense/turrets/ChargeTurret.java

```diff
@@ -28,7 +32,8 @@ public void shoot(BulletType ammo){
 
             tr.trns(rotation, size * tilesize / 2f);
             chargeBeginEffect.at(x + tr.x, y + tr.y, rotation);
-
+            chargeSound.at(x + tr.x, y + tr.y, Mathf.random(0.8f, 1f));
```

Comment:
Pitch should not be random.

Reply:
Ok, I'll change that, though I did copy the code for the shootSound.